doc/dev-guide.rst

```diff
@@ -0,0 +1,113 @@
+.. encoding: utf-8
+
+Developer's guide
```

Comment:
It seems to me that this can be merged with "Quick start". I didn't really understand the sence of the "Developers guide". Up to you.

Reply:
Well, the main reason is that I have followed the existing structure. "Developers guide" is more about complicated questions rather than "How to connect". But I think this separation is ok, so if you give me a choice, I leave it as is.